### ISSUE TITLE
[Feature Request] Add banner patterns when installed with Archeology Plus

### ISSUE THREAD

**Issue by Ultimushadow** [Darkhax-Minecraft/Archaeology-Banners#1]
Hey there, I love the mod and the modularity of the patterns from this. I have another mod installed in my game called Archeology Plus which adds its own sherds and thought it was a shame there wouldn't be any banner patterns for those, so I made some! I plan on modding it into my game by piggybacking off your mod anyways, but I figured I might as well through this here in case you wanted to add a compatibility feature! They are.. not great, and it's just the 16x16 of the icon atm, but I figured it might be worth throwing in if you don't want to go all out on it since it's for another mod.

[archeology_plus_banner_template_starters.zip](https://github.com/Darkhax-Minecraft/Archaeology-Banners/files/12017746/archeology_plus_banner_template_starters.zip)

**Comment by Darkhax** [comment]
Thanks for the request and letting me know about this mod. I am interested in adding support for modded sherd designs but it may take a little bit to create all the new designs. The designs for the base mod are based on the decorated pot texture rather than the item sprites because the designs on those are a bit cleaner and have more details.

**Comment by Ultimushadow** [comment]
Yeah if you look through the zip most of those are based off the pot textures from Archeology Plus, I messed with a couple because some of them had an odd numbered pixel length, I didn't do a lot of the multiple segments like you have for the vanilla sherds though so there's definitely some polish needed

**Comment by Darkhax** [comment]
I've looked into this a lot more and have finished textures and sub variants for all of the Archeology Plus sherds. I've also made a few internal tools to help me work on these types of textures which should make it easier to do compatibility for other mods moving forward. This also means official compatibility should come soon.

Initially I planned to release support for Archeology Plus built in to the base mod, but I've decided to distribute it as an optional addon mod instead for performance reasons. I noticed that Minecraft will load all pot textures and related localization entries even if the other mod is loaded, meaning all users of the mod would be wasting some of their RAM and GPU RAM on data that will never be used. There are a few solutions to this, but after looking into these solutions and talking with devs of projects that have tried them this is the most reliable solution that will work with all mod loaders.
